test/explorer.test.ts

```diff
@@ -6,8 +7,8 @@ import * as fakes from './fakes';
 
 import { Host } from '../src/host';
 import { Shell, ShellResult } from '../src/shell';
-import { FS } from '../src/fs';
-import { create as explorerCreate } from '../src/explorer';
+import { fs } from '../src/fs';
```

Comment:
It looks like the original import wasn't used, but I'd still rather not import the implementation instead of the interface.  `../src/fs` is meant to be an abstraction layer that can be injected into a context.  Looking at the usage, you're not injecting fake responses to filesystem calls, so you can just call the system 'fs' package, e.g. `import * as sysfs from 'fs';` - don't bother with the abstraction as the filesystem call is part of the test implementation.